styles/stylesheet.css

```diff
@@ -0,0 +1,378 @@
+* {
+    margin: 0;
+    padding: 0;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+h2 {
+    margin-top: 3%;
+}
+
+
+
+h2,
+h3,
+h4,
+.dogs img,
+.dogs p,
+.all-dogs p {
+    margin-bottom: 5%;
+}
+
+.blog1 h2,
+.blog2 h2,
+.blog3 h2,
+.blog1 p,
+.blog2 p,
+.blog3 p {
+    margin-top: 0;
+    margin-bottom: 2%;
+}
```

Comment:
Each blog should have a common class, that way you won't have to use .blog2, .blog3, etc....just a `.blog` should select all of them. Think of how you could adjust your html to use this.

```
.blog p,
.blog h2 {
    margin-top: 0;
    margin-bottom: 2%;
}```